plugins/modules/ipadnsrecord.py

```diff
@@ -289,7 +337,7 @@
       loc_lon_dir:
         description: LOC Direction Longitude
         required: false
-        choices: ["E", "W"]
+        type: str
```

Comment:
Shouldn't we keep the choices "E" and "W", as only them should be used (RFC 1876)

Reply:
oh... my bad. They should be on the argument_spec. Please add to both.